### DISCUSSION TITLE
fetch rpc

### DISCUSSION THREAD

**Discussion by Walterhart** [supabase/supabase#10406] [Questions] [answered]
When I use sql editor below for function I get a result, but when I try it in java script no data is being fetched and displayed in the console log. What am I doing wrong?

Sql editor:
![image](https://user-images.githubusercontent.com/70658679/202767020-faf65c71-36a8-4ce5-bc33-95f7da087e7e.png)
note: the function is a simple select a netflix.table and return query

javascript:
![image](https://user-images.githubusercontent.com/70658679/202767443-1017877d-1f0d-449a-bff5-384faac60c93.png)

**Comment by GaryAustin1** [comment, marked as answer]
This usually means you are not meeting RLS policies for the tables the function is using.

**Reply by Walterhart** [reply]
The RLS policy is off for the netflix_table.

**Reply by GaryAustin1** [reply]
Was the table created by Prisma?  If so you have to restore privileges in certain cases.
Are you seeing in errors in postgres logs from the dashboard?
When you say no data, do you mean empty array?  Also any errors?

**Reply by Walterhart** [reply]
Actually you was correct. On table it said no active policy. But when I clicked and check it showed policy enabled. Thanks for the help
